clusterloader2/pkg/measurement/common/scheduling_throughput.go

```diff
@@ -167,4 +171,5 @@ type schedulingThroughput struct {
 	Perc50  float64 `json:"perc50"`
 	Perc90  float64 `json:"perc90"`
 	Perc99  float64 `json:"perc99"`
+	Max     float64 `json:"max"`
```

Comment:
Are we going to delete the Average field as well?

Reply:
I'm not sure about it - I know that there is #1083 about broken metric.

Let's leave it as it for now without touching this issue - I will either fix avg or remove it, however it's not a critical part of solving Scheduler Throughput meassurment.

Reply:
Sound good as long as we have in on the radar

Reply:
After second though - removed Avg as we are no using it. It there will be a FR to keep it, then we will need to figure out how should it behave.

Resolving #1083